astropy/utils/iers/iers.py

```diff
@@ -484,7 +476,7 @@ def _combine_a_b_columns(cls, iers_a):
         # IERS B values in the table are consistent with the true ones.
         table = cls._substitute_iers_b(table)
 
-        q_where = cls._quantity_where if NUMPY_LT_1_17 else np.where
+        q_where = np.where
```

Comment:
Ideally, we'd just replace `q_where` with `np.where`... (sorry to have set up this extra work for you!)